make_group_file.py

```diff
@@ -29,14 +29,85 @@
 import click
 import logging
 
-import math
+from typing import List
```

Comment:
depending on how urgent this PR is, you can remove this. Type hinting hasn't required an imported `List` (Upper-case) for a few versions now. You can just use the lower-case `list` in type hints.

Reply:
welcome back! Not super urgent, thanks just made the change!

Reply:
testing just in case: https://batch.hail.populationgenomics.org.au/batches/484392